README.md

```diff
@@ -1,27 +1,29 @@
 # pgsql2
-Implementation of a proposed asynchronous SQL specification for PostgreSQL
 
-## What problem does this library solve
+An implementation of [ADBA/jdk.sql2](http://cr.openjdk.java.net/%7Elancea/8188051/apidoc/jdk/incubator/sql2/package-summary.html), a proposed asynchronous SQL specification, for PostgreSQL.
```

Comment:
I am a bit unsure what the name for this proposed standard is, the link is the best I could find (that wasn't Oracle marketing), please correct it if it is not the best one.

Reply:
I think sql2 was an internal catalog name where they put the first version of the source code, and that has lived on, ADBA is the more official name.

We are in the process of renaming the project itself to pgAdba.